src/Components/Scenes/RequestAssets/MakeRequests/MakeRequests.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Button } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import { useHistory } from 'react-router-dom';
+import SnackBar from '../../../Presentational/RequestAssets/SnackBar';
+
+const tableStyles = makeStyles(theme => ({
+  button: {
+    textTransform: 'none',
+    fontWeight: 'bold',
+    color: theme.palette.common.white,
+    margin: 8
+  }
+}));
+
+const MakeRequests = props => {
+  var {
```

Comment:
nitpick: I tend to stay away from using `var` because they aren't strictly enforced. Maybe `const` instead?

Reply:
I prefer to use `const` or `let`. But I was testing something. Then I forget it there.  I will going to change it.